nemoguardrails/library/autoalign/README.md

```diff
@@ -571,6 +579,13 @@ rails:
 Specify the factcheck endpoint the parameters section of autoalign's config.
 Then, you have to call the corresponding subflows for factcheck guardrails.
 
+In the guardrails config for factcheck you can toggle "verify_response" flag
+which will enable(true) / disable (false) additional processing of LLM Response.
+This processing ensures that only relevant LLM responses undergo fact-checking
+and responses like greetings ('Hi', 'Hello' etc.) do not go through fact-checking
+process.
+
```

Comment:
can you also add:

"Note that the verify_response is set to False by default as it requires additional computation, and we encourage users to determine which LLM responses should go through AutoAlign fact checking whenever possible."